source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -120,6 +121,9 @@ def _extend_journey(pt_journey, fallback_dp, fallback_extremity):
     pt_journey.duration += aligned_fallback.journeys[0].duration
     pt_journey.durations.total = pt_journey.duration
     pt_journey.durations.walking += aligned_fallback.journeys[0].durations.walking
+    pt_journey.durations.bike += aligned_fallback.journeys[0].durations.bike
+    pt_journey.durations.car += aligned_fallback.journeys[0].durations.car
+
```

Comment:
What about crowfly? Is it also necessary in https://github.com/CanalTP/navitia/blob/43e3fd6a1e4247b311954125fdb5d059932097de/source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py#L53

Reply:
In the specific case where the crowfly has a duration, it is associated to walking. Bike and car duration aren't involved.